server.js

```diff
@@ -1,34 +1,18 @@
 const express = require("express");
 const home = require("./routes/home.js");
+const deletePost = require("./routes/deletePost.js")
 
 const server = express();
 
 const staticHandler = express.static("public");
 server.use(staticHandler);
 
+const bodyParser = express.urlencoded({ extended: false });
```

Comment:
I still need to wrap my head around this, but have faith you know what you're doing!